### ISSUE TITLE
Neshawk causes glitches in Saint Seiya Ougon Densetsu Kanketsu-hen

### ISSUE THREAD

**Issue by GoogleCodeExporter** [TASVideos/BizHawk#352]
```
What steps will reproduce the problem?
1. Play Saint Seiya Ougon Densetsu Kanketsu-hen.
2. Use Last Dragon attack.
3.

What is the expected output? What do you see instead?
Menu bars on top and bottom should remain gray. They turn black instead.

What version of the product are you using? On what operating system?
1.9.1

Please provide any additional information below.


```

Original issue reported on code.google.com by `Reinc...@gmail.com` on 26 Dec 2014 at 5:45

Attachments:
* [Saint Seiya - Ougon Densetsu Kanketsu Hen (J).nes](https://storage.googleapis.com/google-code-attachments/bizhawk/issue-352/comment-0/Saint Seiya - Ougon Densetsu Kanketsu Hen (J).nes)
* [how it should be.State](https://storage.googleapis.com/google-code-attachments/bizhawk/issue-352/comment-0/how it should be.State)
* [how it is.State](https://storage.googleapis.com/google-code-attachments/bizhawk/issue-352/comment-0/how it is.State)

**Comment by alyosha-tas** [comment]
This bug is resolved by proper DMA timing following this nesdev wiki article

http://wiki.nesdev.com/w/index.php/PPU_OAM

Just need to track even and odd frames and set DMA countdown to 513 or 514 accordingly.

However I presently don't know how best to do this, it seems like it needs to be part of SyncState.

**Comment by hegyak** [comment]
Was this fixed?  I know a LOT of NES timing fixes were implemented recently.

**Comment by alyosha-tas** [comment]
Sadly, no, it wasn't fixed.

This is amongst the more difficult remaining bugs, it will probably be fixed last.

**Comment by alyosha-tas** [comment]
Finally resolved with PR #662
